cmake/ConfigureChecks.cmake

```diff
@@ -158,25 +158,26 @@ int main () {
 "
   TPARM_TAKES_VARARGS
 )
-IF(NOT TPARM_TAKES_VARARGS)
-  CHECK_CXX_SOURCE_COMPILES("
-${TPARM_INCLUDES}
+
+CHECK_CXX_SOURCE_COMPILES("
 #define TPARM_VARARGS
```

Comment:
Is there any harm in just defining this and not even trying the version without it?

Reply:
For posterity, the question was answered by faho:

> ... this makes it so we don't use our awful "solaris kludge" on NetBSD, instead relying on the actual working tparm function

Reply:
I think you have the wrong comment.

This one's about whether just always defining TPARM_VARARGS would cause any issues.

Then we could remove the distinction about "TPARM_TAKES_VARARGS" and "TPARM_TAKES_VARARGS_WITH_VARARGS".

Reply:
Ah, OK- yeah, that does sound simpler and likely to do the right thing.